EXTENSIONS_CSS_MAP

```diff
@@ -0,0 +1 @@
p-story-info-dialog","amp-story-share","amp-story-share-menu","amp-story-system-layer"]},"amp-story-1.0":{"name":"amp-story","version":"1.0","hasCss":true,"cssBinaries":["amp-story-bookend","amp-story-tooltip","amp-story-consent","amp-story-hint","amp-story-unsupported-browser-layer","amp-story-viewport-warning-layer","amp-story-info-dialog","amp-story-share","amp-story-share-menu","amp-story-system-layer"]},"amp-story-auto-ads-0.1":{"name":"amp-story-auto-ads","version":"0.1","hasCss":true},"amp-selector-0.1":{"name":"amp-selector","version":"0.1","hasCss":true},"amp-web-push-0.1":{"name":"amp-web-push","version":"0.1","hasCss":true},"amp-wistia-player-0.1":{"name":"amp-wistia-player","version":"0.1","hasCss":false},"amp-position-observer-0.1":{"name":"amp-position-observer","version":"0.1","hasCss":false},"amp-orientation-observer-0.1":{"name":"amp-orientation-observer","version":"0.1","hasCss":false},"amp-date-picker-0.1":{"name":"amp-date-picker","version":"0.1","hasCss":true},"amp-image-viewer-0.1":{"name":"amp-image-viewer","version":"0.1","hasCss":true},"amp-subscriptions-0.1":{"name":"amp-subscriptions","version":"0.1","hasCss":true},"amp-subscriptions-google-0.1":{"name":"amp-subscriptions-google","version":"0.1","hasCss":true},"amp-pan-zoom-0.1":{"name":"amp-pan-zoom","version":"0.1","hasCss":true},"amp-recaptcha-input-0.1":{"name":"amp-recaptcha-input","version":"0.1","hasCss":true},"amp-slides-0.1":{"name":"amp-slides","version":"0.1","hasCss":false},"amp-social-share-0.1":{"name":"amp-social-share","version":"0.1","hasCss":true},"amp-timeago-0.1":{"name":"amp-timeago","version":"0.1","hasCss":false},"amp-twitter-0.1":{"name":"amp-twitter","version":"0.1","hasCss":true},"amp-user-notification-0.1":{"name":"amp-user-notification","version":"0.1","hasCss":true},"amp-vimeo-0.1":{"name":"amp-vimeo","version":"0.1","hasCss":false},"amp-vine-0.1":{"name":"amp-vine","version":"0.1","hasCss":false},"amp-viz-vega-0.1":{"name":"amp-viz-vega","version":"0.1","hasCss":true},"amp-google-vrview-image-0.1":{"name":"amp-google-vrview-image","version":"0.1","hasCss":false},"amp-viewer-integration-0.1":{"name":"amp-viewer-integration","version":"0.1","hasCss":false,"loadPriority":"high"},"amp-video-0.1":{"name":"amp-video","version":"0.1","hasCss":false},"amp-video-docking-0.1":{"name":"amp-video-docking","version":"0.1","hasCss":true},"amp-video-iframe-0.1":{"name":"amp-video-iframe","version":"0.1","hasCss":false},"amp-video-service-0.1":{"name":"amp-video-service","version":"0.1","hasCss":false,"loadPriority":"high"},"amp-viqeo-player-0.1":{"name":"amp-viqeo-player","version":"0.1","hasCss":false},"amp-vk-0.1":{"name":"amp-vk","version":"0.1","hasCss":false},"amp-yotpo-0.1":{"name":"amp-yotpo","version":"0.1","hasCss":false},"amp-youtube-0.1":{"name":"amp-youtube","version":"0.1","hasCss":false},"amp-mowplayer-0.1":{"name":"amp-mowplayer","version":"0.1","hasCss":false},"amp-powr-player-0.1":{"name":"amp-powr-player","version":"0.1","hasCss":false}}
```

Comment:
What is this file?

Reply:
OK. Not part of the build release. Removed.